core-web/libs/ui/src/lib/components/dot-drop-zone/dot-drop-zone.component.ts

```diff
@@ -44,7 +44,7 @@ export class DotDropZoneComponent {
      * Max file size in bytes.
      * See Docs: https://www.dotcms.com/docs/latest/binary-field#FieldVariables
      */
-    @Input() maxFileSize: number;
+    @Input() maxFileSize: number | null = null;
```

Comment:
No change to input signal?

Reply:
I don't recall if this component is used in DOJO. And converted to a web component can be used with a new signal.